### DISCUSSION TITLE
How to fix documentation

### DISCUSSION THREAD

**Discussion by BryanHogan** [motiondivision/motion#2909] [Q&A]
The documentation is somewhat lacking and has some errors, but I can't find a way to submit changes?

For example https://motion.dev/docs/stagger , the first code blocks has one curly bracket too much.

And the quick start page also has lots of room for improvement, it doesn't even mention that you should add the name of the object you try to animate as a class to the element you want to animate.

**Comment by mattgperry** [comment]
The docs aren’t open source so you can add your pain points here and I can look to addressing them. 

> it doesn't even mention that you should add the name of the object you try to animate as a class to the element you want to animate.

I’m not sure what you mean by this. These aren’t names - they’re normal DOM selectors. if you have a class you can target that, or an element name, attribute name etc. I’m not sure it’s within the scope of this page to explain what selectors are though I could add an MDN link for those unfamiliar.
